src/lighteval/config/lighteval_config.py

```diff
@@ -37,9 +39,8 @@
 DEFAULT_GENERATION_SEED = 42
 
 
-@dataclass
-class GenerationArgs:
-    sampler: Optional[Union[str, "SamplerType"]] = None
+class GenerationArgs(BaseModel):
```

Comment:
ooc, what's the advantage of a pydantic basemodel over a dataclass?

Reply:
https://github.com/huggingface/lighteval/blob/ce1dbb5eab6edf2c8a8c297952e97ea3b3067041/src/lighteval/pipeline.py#L158

Reply:
thanks to you both, til!